sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -142,25 +152,26 @@ export interface QueueOptions {
   duplicateDetectionHistoryTimeWindow?: string;
```

Comment:
Documentation for `duplicateDetectionHistoryTimeWindow` should say something about the string format.

Reply:
this can be added as part of #6044 merge

Reply:
Sounds good!